src/kirby/components/tabs/tab-button/tab-button.events.ts

```diff
@@ -0,0 +1 @@
+export const tabClicked = 'tabClicked';
```

Comment:
Should Kirby events be prefixed with `kirby`to avoid clashes? 🤔

Reply:
Good idea